routers/frontend/routes.go

```diff
@@ -221,7 +222,7 @@ func (r *frontendRouter) Register(ctx *gin.Context) {
 		return
 	}
 
-	user, err := r.userService.CreateUser(ctx, name, email, password)
+	user, err := r.userService.CreateUser(ctx, name, email, password, role.Unverified)
```

Comment:
Should this instead use `r.cfg.Auth.DefaultRole`?

Reply:
I only made this change to make sure the code builds correctly. This handler uses v1 of the auth system and will get replaced in the near future. I didn't want to use the config here because then you have to set it up for the tests as well and it's just unnecessary changes.

Reply:
Very good point 👍